parsec/backend/asgi/rpc.py

```diff
@@ -63,15 +93,194 @@ async def anonymous_api(raw_organization_id: str):
         except OrganizationAlreadyExistsError:
             pass
 
-    # Retreive command
+    # Retrieve command
     client_ctx = AnonymousClientContext(organization_id)
     try:
         if not isinstance(cmd, str):
-            raise KeyError()
+            raise KeyError(f"`cmd` type is not a string but a {type(cmd).__name__}")
         cmd_func = backend.apis[ClientType.ANONYMOUS][cmd]
-    except KeyError:
+    except KeyError as e:
+        logger.warning(f"Anonymous command {cmd} doesn't exist ({e})")
         return rpc_msgpack_rep({"status": "unknown_command"})
 
     # Run command
     rep = await cmd_func(client_ctx, msg)
     return rpc_msgpack_rep(rep)
+
+
+@rpc_blueprint.route("/authenticated/<raw_organization_id>", methods=["POST"])
+async def authenticated_api(raw_organization_id: str):
+    backend: BackendApp = g.backend
+    logger = current_app.logger.getChild(authenticated_api.__name__)
+
+    logger.debug(
```

Comment:
structlog allow us to do structured logging, so we can do instead:
```python
logger.debug("Received authenticated request", org=raw_organization_id, device=request.headers.get("Author", "<missing>"))
```

Reply:
so according to [hypercorn documentation](https://pgjones.gitlab.io/hypercorn/how_to_guides/logging.html#how-to-log), it should be enough to use the `{Header}i` format in the access log format: https://github.com/Scille/parsec-cloud/blob/890490c21d801fdc2090ec2a4ce4d3ecd159992e/parsec/backend/asgi/__init__.py#L126

Given we already have the organization id in the endpoint, we only need to provide the device id by doing something like:
` "access_log_format": "%(h)s %(r)s %(s)s %(b)so %(D)sus (device_id:%({Author}i)s)",`

However this is not great given we don't return the actual DeviceID but a base64 of the DeviceID :(
On top of that all request unrelated to the authenticated route will have an ugly `(device_id:)` in the log...

The thing is we really want to have the device id in the log given it's really useful we looking at the log to see the different queries a given client has done...

Reply:
This is getting too complex for this PR, can you:
- open an "provide device id in server access log for authenticated http request" issue
- remove all the logs in here (we don't need that level of debug)

Reply:
I've removed all the logs, I let you open the issue because you have all the details

Reply:
I've created #2953

Reply:
:+1: